public/app.js

```diff
@@ -1,6 +1,8 @@
 angular.module('app', [
   'app.navbar',
   'app.createEvent',
+  'app.signupLogin',
+  'app.factories',
```

Comment:
We might want to just leave it in one factory until we refactor to events, users, and the eventsAndUsers factories